### ISSUE TITLE
AMP not blocked

### ISSUE THREAD

**Issue by ghost** [bromite/bromite#1921]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

99.0.4844.77

### Device architecture

armeabi-v7a

### Android version

8.1

### Device model

`Samsung Galaxy J2 Core (SM-J260G)`

### Changed flags

disable-webgl, offline-indicator-v2, incognito-screenshot

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No.

### Describe the bug

`AMP` is not blocked on Bromite, Even with the [AMP blocking patch](https://github.com/bromite/bromite/blob/master/build/patches/Inject-scripts-for-AMP-tracking-ads-and-video.patch).

### Steps to reproduce the bug

1. Go to Google search 
2. Search Something
3. Click on a site that supports `AMP`

### Expected behavior

It should block `AMP` as there's a [patch](https://github.com/bromite/bromite/blob/master/build/patches/Inject-scripts-for-AMP-tracking-ads-and-video.patch) that should do that.

### Screenshots

![Screenshot_20220324-025656](https://user-images.githubusercontent.com/95953932/159799938-1105870c-09eb-4029-b930-94846a220f09.jpg)

**Comment by JustAnUserLol** [comment]
I workarounded this by using a [userscript](https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.user.js).

**Comment by csagan5** [comment]
Fixed in `100.0.4896.57`.

**Comment by csagan5** [comment]
That user script looks like a good alternative, perhaps the embedded one should be removed.

**Comment by alxlg** [comment]
@csagan5 how does one revert this? When using Google Images there is no way to open the original address and the AMP overlay is empty. Also, despite what some say, I don't think AMP being a privacy issue. It's not even needed for SEO, indeed Google just requires a site to be loaded fast for better placement.

**Comment by csagan5** [comment]
> how does one revert this?

Revert what?

> When using Google Images there is no way to open the original address and the AMP overlay is empty

Please submit a separate new issue.
